spec/asyncapi.md

```diff
@@ -1607,20 +1607,20 @@ For this specification, reference resolution is done as defined by the JSON Refe
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="referenceRef"></a>$ref | `string` | **REQUIRED.** The reference string.
```

Comment:
The description of the Reference Object above should be updated too, to reflect that it's not strictly oriented to JSON format only and it may change in the future. Otherwise, it would be a breaking change.